docs/rundown.rst

```diff
@@ -65,6 +98,23 @@ and two small playbooks under a ``playbooks`` directory.
    take the project root directory as their last argument, thus we will simply
    use ``.`` to indicate the current Working Directory.
 
+Adding a GPG key to AWX or Ansible Automation Controller:
+=========================================================
+
+In the command line, run the following commands:
+``$ gpg --list-keys``
+``$ gpg --export --armour <key fingerprint> > my_public_key.asc``
+
+
+In AWX/Automation Controller, click “Credentials."
+From there, click “Add”.
+For “Name” name your credential something you will recognize.
+For “Credential Type” select  “GPG Public Key."
+
+Click "Browse" to navigate to and select the file that you created earlier. Great work! You're ready to start verifying signed project content. 
```

Comment:
Might be obvious but for completion might as well mention "click the save button" to finish.

Also, maybe the end should be something like "This credential can now be selected in Project settings. Once selected, content verification will automatically take place on future project syncs."

This can be reworded in whatever way makes sense.